app/helpers/link_helper.rb

```diff
@@ -0,0 +1,35 @@
+module LinkHelper
+  EXTERNAL_LINK_CLASS = 'usa-link--external'.freeze
+
+  def new_window_link_to(name = nil, options = nil, html_options = nil, &block)
+    if block_given?
+      html_options = options
+      options = name
+      name = block
+    end
+    options ||= {}
+
+    html_options = convert_options_to_data_attributes(options, html_options)
```

Comment:
I guess it could be good to normalize before we operate on any of the properties, but since `link_to` will be doing this (again) anyways to the options we pass, it doesn't seem strictly necessary.

Reply:
yep, removed in a73e236